src/main/cpp/RobotContainer.cpp

```diff
@@ -66,14 +64,18 @@ void RobotContainer::ConfigureBindings() {
                     return m_drive.ResetOdometry(Pose2d());
                   }).ToPtr());
 
-  m_operator.X().OnTrue(IntakeCone(&m_gripper).ToPtr());
-  m_operator.X().OnFalse(
-      InstantCommand([this]() { return m_gripper.Retract(); }, {&m_gripper}).ToPtr());
-  m_operator.Y().OnTrue(IntakeCube(&m_gripper).ToPtr());
-  m_operator.Y().OnFalse(
-      InstantCommand([this]() { return m_gripper.Retract(); }, {&m_gripper}).ToPtr());
-  m_operator.A().OnTrue(
-      InstantCommand([this]() { return m_gripper.EjectGamePiece(); }, {&m_gripper}).ToPtr());
-  m_operator.A().OnFalse(
-      InstantCommand([this]() { return m_gripper.Retract(); }, {&m_gripper}).ToPtr());
+  // m_operator.X().OnTrue(IntakeCone(&m_gripper).ToPtr());
```

Comment:
Where is the OI stuff now?

Reply:
We still need to work out the arm control system, but that seemed like a separate PR.

Reply:
Makes sense, we should work on that today